lib/terraform/runner.rb

```diff
@@ -134,11 +131,11 @@ def create_stack_job(
 
         # TODO: use tags,env_vars
         payload = {
-          :cloud_providers => credentials,
+          :cloud_providers => to_cloud_providers_conn_params(credentials),
```

Comment:
See tenant comment above.  I wonder if tenant_id should be provider_tenant_name if it's a name or provider_tenant_id if it's a number.  We have tenant_id columns on many tables so this could be easily confused.

Reply:
renamed the `tenant` field as `provider_tenant`